strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/providers/layout/NugetLayoutProvider.java

```diff
@@ -112,9 +107,9 @@ private String toBase64(byte[] digest)
     public void rebuildMetadata(String storageId,
                                 String repositoryId,
                                 String basePath)
-            throws IOException,
-                   NoSuchAlgorithmException,
-                   XmlPullParserException
+        throws IOException,
```

Comment:
I'm not sure why your IDE reformatted this block (here and in several places below) without there being any actual changes.

Reply:
we discussed this while ago, the thing is that I cant customize this part of Eclipse formatter, this variant is the best that I can do :(

Reply:
fixed